spec/index.md

```diff
@@ -335,163 +325,131 @@ Each dimension is represented by a key in the `dimensions` object. The object ha
 }
 ```
 
-## Examples
```

Comment:
I just moved them to the bottom, so they didn't break the flow. The dimension type stuff really follows on immediately from defining what dimensions are, so it was better not to have a big chunk of examples first.

(It's possible I broke something - I don't know how to build the page, so I don't actually see anything in the examples section.)